packages/block-editor/src/components/block-edit/index.js

```diff
@@ -1,52 +1,17 @@
-/**
- * WordPress dependencies
- */
-import { Component } from '@wordpress/element';
-
 /**
  * Internal dependencies
  */
 import Edit from './edit';
 import { BlockEditContextProvider } from './context';
 
-class BlockEdit extends Component {
-	constructor( props ) {
-		super( props );
-
-		this.setFocusedElement = this.setFocusedElement.bind( this );
-
-		this.state = {
-			focusedElement: null,
-			setFocusedElement: this.setFocusedElement,
-		};
-	}
-
-	setFocusedElement( focusedElement ) {
```

Comment:
@gziolo Would love if you could test as well, as you first introduced in `setFocusedElement` #6419.